docs/docs/workflows/s3proxy.md

```diff
@@ -0,0 +1,56 @@
+# Install s3proxy
+
+Constellation includes a transparent encryption proxy for [AWS S3](https://aws.amazon.com/de/s3/).
+s3proxy will encrypt/decrypt objects when you send/retrieve objects to/from S3, without making changes to your application necessary.
+
+## Limitations
+
+:::caution
+Using s3proxy outside Constellation is insecure as the connection between the key management service (KMS) and s3proxy is protected by Constellation's WireGuard VPN.
+The VPN is a feature of Constellation and won't be present by default in other environments.
+:::
+
+These limitations will be removed with future iterations of s3proxy.
+
+- Only `PutObject` and `GetObject` requests are encrypted/decrypted by s3proxy.
+That means other requests are forwarded to S3 as-is.
+For example, objects uploaded with `CreateMultipartUpload` aren't encrypted.
+- Using the [Range](https://docs.aws.amazon.com/AmazonS3/latest/API/API_GetObject.html#API_GetObject_RequestSyntax) header on `GetObject` is currently not supported and will result in an error.
+
+If you want to use s3proxy but these limitations stop you from doing so, please consider [opening](https://github.com/edgelesssys/constellation/issues/new?assignees=&labels=&projects=&template=feature_request.yml) an issue.
+
+## Deployment
+
+- `git clone git@github.com:edgelesssys/constellation.git && cd constellation`
+- Replace the values named `replaceme` in `s3proxy/deploy/deployment-s3proxy.yaml` with valid AWS credentials. These credentials are used by s3proxy to access your S3 buckets.
+- `kubectl apply -f s3proxy/deploy/deployment-s3proxy.yaml`
+
+s3proxy is now deployed.
+If you want to run a demo application you can checkout the [Deploy Filestash](https://github.com/edgelesssys/constellation/tree/main/s3proxy/deploy#deploying-filestash) instructions to deploy Filestash.
+Filestash is a web frontend for different storage backends, including S3.
```

Comment:
add this to the examples section of the docs and try making it a bit prettier like the existing examples